Web/starter_project_vanguard/pages/_app.tsx

```diff
@@ -1,6 +1,5 @@
 import '@/styles/globals.css'
 import type { AppProps } from 'next/app'
-
```

Comment:
please let us avoid changes that have nothing to do with the feature (if it is something that needs to happen, it can be a different PR)

i would say bring back this change